release.sh

```diff
@@ -0,0 +1,84 @@
+#!/bin/bash
+#! Author: Kang Lin <kl222@126.com>
+
+set -e
+
+SOURCE_DIR=`pwd`
+
+if [ -n "$1" ]; then
+    VERSION=`git describe --tags`
+    if [ -z "$VERSION" ]; then
+        VERSION=`git rev-parse HEAD`
+    fi
+
+    if [ -n "$2" ]; then
+        MESSAGE=$2
+    fi
+    PRE_TAG=`git tag --sort=-taggerdate | head -n 1`
+    echo "Current verion: $VERSION, current tag: $PRE_TAG. The version to will be set: $1 $MESSAGE"
```

Comment:
Not required, just run the script

Reply:
This is required. Used to prompt you for preparation, check that you entered the correct version number.